src/_tests_/EventContainer.test.js

```diff
@@ -31,6 +45,97 @@ describe('EventContainer', () => {
     expect(wrapper).toMatchSnapshot()
   });
 
+  describe('filterEvents', () => {
```

Comment:
These tests look thorough. It's really interesting to see the difference between FE and BE tests!

Reply:
Thanks! Yeah, testing what is shown on the DOM can be tricky. As you'll see in all our component test files, we have these `snapshot` tests that check if what we are rendering on the page matches what is expected by the tests we set up.